chapter_13_database_layer_validation.asciidoc

```diff
@@ -75,10 +72,6 @@ use to validate input, but you're always going to use the same database.
 
 ==== The self.assertRaises Context Manager
 
-// SEBASTIAN: This seems to come out of the blue, at least a bit.
```

Comment:
At first glance I agreed, but then I re-read the chapter from the beginning and it feels it's been linked properly by the second paragraph above.